src/etc/inc/plugins.inc.d/unbound.inc

```diff
@@ -297,6 +326,38 @@ EOD;
 
     $so_reuseport = empty(system_sysctl_get()['net.inet.rss.enabled']) ? 'yes' : 'no';
 
+    $dohdot_settings = '';
+    if (isset($config['unbound']['enable_doh']) || isset($config['unbound']['enable_dot'])) {
+        $cert =& lookup_cert($config['unbound']['dohdot_cert']);
+	$chain = [];
```

Comment:
Indentation on this and some following lines should be improved IMHO. From indentation I think the lines 356 & 357 are inside the `if` block. From the parenthesis these lines are not part of the `if`block.

Reply:
Fixed